client/inbox-panel/utils.js

```diff
@@ -42,6 +43,26 @@ export function hasValidNotes( notes ) {
 	return validNotes.length > 0;
 }
 
+/**
+ * Truncates array of text characters.
+ *
+ * @param {Array} letters The letter array to truncate.
+ * @param {number} limit number of characters to limit to
+ * @param {string} separator The separator string to truncate to.
+ */
+const truncate = ( letters, limit, separator = ' ' ) => {
```

Comment:
I think it'll be great to add a unit test for this function

Reply:
Sounds good. Added in https://github.com/woocommerce/woocommerce-admin/pull/8404/commits/ae4ec6c504fac7504bc62eae5b3ea96d44c55a8a.